2093. Cells stuck running

maxyeg opened the pull request:
Fix for issue #2093

mnavasiolava committed:
more plot legend improvements #1617 (cursor should be hand for dragging not I-bar for editing)

kpaharelau committed:
fix plots need more lines #2029

Algorithm for calculation time increment has been fully changed. Use notebook for tests - https://www.dropbox.com/s/xwiz4max5htq83e/2029.bkr?dl=0

mnavasiolava committed:
more plot legend improvements #1617 (option to omit checkboxes)

kpaharelau committed:
fix plots need more lines #2029

I made predefined time values more detailed. It should solve all problems with gridlines

Commit:
1878. ggmap crashes R backend

kpaharelau committed:
fix plots need more lines #2029

I made predefined time an values more detailed and decreased desired step for time axe to 105 from 150.

scottdraves commented 19 hours ago
Thanks.
This is a step in the right direction. But still, the lines are never closer than about 104 pixels.
can you make them go down to 50?

This is not a good idea. Time labels may overlap in some cases.

Commit:
1878. ggmap crashes R backend

kpaharelau committed:
fix plots need more lines #2029

spot [5:49 PM]
3.8 should be 3.75 FIXED

kpaharelau committed:
fix plots need more lines #2029

x axis needs months #2042 - fix

mnavasiolava committed:
more plot legend improvements #1617 (horizontal legend)

Commit:
2093. Cells stuck running

samccone commented on core/src/main/web/app/mainapp/mainapp.js:
```diff
@@ -208,6 +208,19 @@
 
+            //Clear previous evaluation in progress state.
+            //Set state to Error with message "Interrupted."
+            if (notebookModel && notebookModel.cells) {
```
Thoughts on moving this into the`bk.notebookCellModelManager` helper?

Commit:
2093. Cells stuck running.
Move clear logic from load to save.
Place it in notebookCellModelManager

ildipo committed:
pass system preferences to output display rather than fetching them.
This is needed for using output displays inside publication server.

samccone committed:
Add autotranslation tests

ildipo committed:
Update embeddable assets and fix some paths

ildipo committed:
one more update

scottdraves commented on core/src/main/web/app/mainapp/services/notebookcellmodelmanager.js:
```diff
@@ -453,6 +453,18 @@
+            if (currentCell && currentCell.output && currentCell.output.result
+                && currentCell.output.result.innertype === 'Progress') {
+              currentCell.output.result.innertype = 'Error';
+              currentCell.output.result.object = 'Interrupted.'
```
error message should read:

```
Interrupted, saved while running
```

ildipo committed:
reworked bkHelper error handling

ildipo committed:
forgot to refactor one function

samccone committed:
tests: Fix grooxy flakey test

samccone committed:
tests: Increase delay for notebook load time

Commit:
2093. Cells stuck running.
Change msg to 'Interrupted, saved while running'.
Do not alter the current notebook model. Make changes only to the model of the notebook that is being saved to disk.
